### PR TITLE
Update resources spec to support property aliases.

### PR DESCRIPTION
**Pull request by TianoMS**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@TianoMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by vivsriaus** [comment]
LGTM

File: arm-resources/resources/2016-02-01/swagger/resources.json [hunk 1]
```diff
@@ -1647,6 +1671,36 @@
...
+        "path": {
+          "type": "string",
+          "description": "Gets or sets the path."
+        },
+        "apiVersions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the api versions."
```
Comment (devigned) [inline arm-resources/resources/2016-02-01/swagger/resources.json]: Please don't add "Gets or sets" to the description. The generator will take care of this for you in each languages' way.

Please apply this to other property descriptions.

**Comment by devigned** [comment]
@vivsriaus what is the purpose of the property alias? Could you elaborate on that a bit?

**Comment by vivsriaus** [comment]
@devigned From here: https://azure.microsoft.com/en-us/documentation/articles/resource-manager-policy/
Property alias is a name that can be used in a policy definition to access the resource type specific properties, such as settings, and skus. It works across all API versions where the property exists.

@ravbhatnagar or @cyl3392207 can elaborate more on this, if needed

File: arm-resources/resources/2016-02-01/swagger/resources.json (hunk 1 above)
Comment (TianoMS) [inline arm-resources/resources/2016-02-01/swagger/resources.json]: Makes sense. I'll update all our descriptions in a separate task.

**Comment by TianoMS** [comment]
Can we get this PR merged?

**Pull request merged**